superbench/common/utils/nvidia_helper.py

```diff
@@ -5,18 +5,185 @@
 
 import py3nvml.py3nvml as nvml
 
+from superbench.common.utils import logger
+from superbench.common.utils import run_command
+
+nvml.nvmlInit()
```

Comment:
It's a little bit weird to init it globally.

Reply:
It's quite like we should provide a class on this. It can init nvml, handle , and util.

Reply:
We can use this class as logger.

Reply:
Done, and abstract the utility as a class.